### PR TITLE
start code

### PR DESCRIPTION
**Pull request by AsselAlAssel**

### CODE REVIEW

File: src/components/DropList.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import "./DropList.css"
+
+const DropList = (props) => {
+
+    const dropListItemClickHandler = (event) => {
```
Comment (ramitheeb) [inline src/components/DropList.js]: call it handelDropListItemClick -> function name should be a verb

File: src/components/DropList.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        const index = event.target.getAttribute("data-index");
+        props.onChangeDropListValue(props.countriesData[index]);
+        props.onClickOnAnyDropListItem();
```
Comment (ramitheeb) [inline src/components/DropList.js:8]: onClickOnAnyDropListItem => onDropListItemClick
onChangeDropListValue  => onDropListValueChange

File: src/components/DropListButton.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import "./DropListButton.css"
+import dropDown from "../assets/images/dropDown.svg"
+
+const DropListButton = (props) => {
+    return (
+        <button onClick={() => { props.onChangeVisibilityOfDopList() }}
```
Comment (ramitheeb) [inline src/components/DropListButton.js]: `<button onClick={props.onChangeVisibilityOfDopList}` 
should be enough

File: src/components/use-fetch.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+
+const useFetchData = (URL) => {
```
Comment (ramitheeb) [inline src/components/use-fetch.js:3]: great job <3

File: src/components/DropListInput.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+import "./DropListInput.css"
+
+const DropListInput = props => {
+    const [inputValue, setInputValue] = useState("");
+
+
+    const changePhoneInputHandler = (event) => {
+        const EnteredValue = event.target.value;
```
Comment (ramitheeb) [inline src/components/DropListInput.js]: enteredValue with small e

File: src/components/DropListInput.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        const IsInputValueHasLengthMorThan3AndLastCharIsHyphen = EnteredValue.length >= 3 && EnteredValue.at(-1) === "-";
```
Comment (ramitheeb) [inline src/components/DropListInput.js]: let's discuss this

File: src/components/DropList.js (hunk 1 above)
Comment (AsselAlAssel) [reply src/components/DropList.js]: done .

File: src/components/DropList.js (hunk 2 above)
Comment (AsselAlAssel) [reply src/components/DropList.js:8]: done

File: src/components/DropListButton.js (hunk 3 above)
Comment (AsselAlAssel) [reply src/components/DropListButton.js]: yes , thx

File: src/components/use-fetch.js (hunk 4 above)
Comment (AsselAlAssel) [reply src/components/use-fetch.js:3]: thanks <3

File: src/components/DropListInput.js (hunk 5 above)
Comment (AsselAlAssel) [reply src/components/DropListInput.js]: Sorry I didn't see that

File: src/components/DropListInput.js (hunk 6 above)
Comment (AsselAlAssel) [reply src/components/DropListInput.js]: I prefer to discuss it in a call.

**Pull request merged**